src/Supportive/DependencyInjection/ServiceContainerBuilder.php

```diff
@@ -129,11 +129,21 @@ private static function loadServices(ContainerBuilder $container, ?SplFileInfo $
             return;
         }
 
-        if (!file_exists($cacheFile->getPathname())
-            && !touch($cacheFile->getPathname())
-            && !is_writable($cacheFile->getPathname())
-        ) {
-            throw CacheFileException::notWritable($cacheFile);
+        if (!file_exists($cacheFile->getPathname())) {
+            $dirname = $cacheFile->getPath() ?: '.';
+
+            if (!is_dir($dirname)
+                && mkdir($dirname.'/', 0777, true)
```

Comment:
The added backslash is neseccary? It just a copy from the baseline class, right?

Reply:
yeah just copied from the baseline not sure why there was a backslash added.